src/app/components/Header/Header.tsx

```diff
@@ -1,17 +1,34 @@
 import React from 'react';
 import 'tailwindcss/tailwind.css';
+import BackButton from '../../components/BackButton/BackButton';
+import ProfileButton from '../../components/ProfileButton/ProfileButton';
 
 type HeaderProps = {
   thumbnail: string;
   name: string;
+  isBackButtonVisible?: boolean;
+  isProfileButtonVisible?: boolean;
 };
 
-function Header({ thumbnail, name }: HeaderProps): JSX.Element {
+function Header({
+  thumbnail,
+  name,
+  isBackButtonVisible = true,
+  isProfileButtonVisible,
+}: HeaderProps): JSX.Element {
   return (
-    <div className="text-secondary flex place-content-center text-3xl border-b-2 border-primary pb-2">
-      <img className="w-7 h-7 mt-1 " src={thumbnail} />
-      {name}
-    </div>
+    <header className="grid grid-cols-3 pb-2 text-3xl border-b-2 text-secondary border-primary">
+      <div className="self-center">
+        {isBackButtonVisible && <BackButton thumbnail="back.png" />}
+      </div>
+      <div className="flex justify-center place-items-center">
+        <img className="w-7 h-7" src={thumbnail} />
+        {name}
```

Comment:
Is the name always the same or does it change based on the url visited?

Reply:
sorry for late reply.. the name is based on the URL